docker-app/qfieldcloud/core/drf_utils.py

```diff
@@ -74,8 +76,8 @@ def _parse_definition(self, definition: str) -> tuple[str, dict[str, str]]:
     def remove_invalid_fields(
         self,
         queryset: QuerySet,
-        fields: list[str],
-        view: viewsets.ModelViewSet,
+        fields: Iterable[str],
```

Comment:
Don't forget to update the docstring too.

Reply:
Updated!